code/modules/urist/gamemodes/scom/sciencedatums.dm

```diff
@@ -113,6 +113,12 @@
 	hidden = 1
 	resources = 800
 
+/datum/scomscience/recipe/sportscar
```

Comment:
take this out for now, I'm tweaking the science system/progression a bit at the moment.